virtualization/Docker/Dockerfile.dev

```diff
@@ -25,7 +25,7 @@ RUN virtualization/Docker/setup_docker_prereqs
 # Install hass component dependencies
 COPY requirements_all.txt requirements_all.txt
 RUN pip3 install --no-cache-dir -r requirements_all.txt && \
-    pip3 install --no-cache-dir mysqlclient psycopg2 uvloop cchardet
+    pip3 install --no-cache-dir mysqlclient psycopg2 cchardet
```

Comment:
Why is uvloop not compatible? Why would aiocoap be tied to an event loop implementation ? 

We can't just remove uvloop, that's a big performance degradation. Also, many users will run under uvloop.